source/Cosmos.IL2CPU/AppAssembler.cs

```diff
@@ -237,14 +237,14 @@ protected void MethodBegin(_MethodInfo aMethod)
                         {
                             METHODLABELNAME = xMethodLabel,
                             IsArgument = false,
-                            INDEXINMETHOD = xLocals[i].Slot,
-                            NAME = xLocals[i].Name,
+                            INDEXINMETHOD = xLocals[i].LocalIndex,
+                            NAME = "",
```

Comment:
Maybe the name should be `"Local" + xLocals[i].LocalIndex`? It was like that before moving to .NET Core I think: https://github.com/CosmosOS/Cosmos/blob/debugger_speed/source/Cosmos.IL2CPU/AppAssembler.cs#L219.